src/main/java/com/marklogic/appdeployer/command/AbstractCommand.java

```diff
@@ -286,7 +284,15 @@ public void setStoreResourceIdsAsCustomTokens(boolean storeResourceIdsAsCustomTo
         this.storeResourceIdsAsCustomTokens = storeResourceIdsAsCustomTokens;
     }
 
-    public void setResourceFilenameFilter(FilenameFilter resourceFilenameFilter) {
+    public void setResourceFilenameFilter(IncrementalFilenameFilter resourceFilenameFilter) {
         this.resourceFilenameFilter = resourceFilenameFilter;
     }
+
+    protected void ignoreHashForFilename(String filename) {
```

Comment:
See my note in DeployRolesCommand below - I don't think this is needed, which would mean you can remove the addFilenameToIgnoreHash method as well.